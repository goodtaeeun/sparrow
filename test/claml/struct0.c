typedef struct {
  int __val[2];
} __fsid_t;

__fsid_t gg;

int f(int x) {
  __fsid_t a;
  a.__val[0] = 1;
  return 1;
}
