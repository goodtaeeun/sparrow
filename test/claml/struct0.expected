/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 1 "struct0.c"
struct __anonstruct_0 {
  int __val[2];
};
#line 3 "struct0.c"
typedef struct __anonstruct_0 __fsid_t;
#line 5
__fsid_t gg;
#line 7 "struct0.c"
int f(int x) {
  __fsid_t a;

  {
#line 9
    a.__val[0] = 1;
#line 10
    return (1);
  }
}
